pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "acih-backend"
+version = "0.1.0"
+description = "Backend for artist-centered image hosting."
+authors = ["Eugene <eugeneturkin@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
Let's use two newlines as a sections separator to make it consistent across the file.

In some section itself we may want some groups to be separated, so for that we'll be able to use one newline **inside** a section and two newlines **between** the sections.